### DISCUSSION TITLE
Are GeoJSON curveRings supported / a thing?

### DISCUSSION THREAD

**Discussion by akrherz** [Toblerity/Fiona#1253] [General]
I'm processing some GeoJSON coming from ESRI, which has [curveRings](https://developers.arcgis.com/python/api-reference/arcgis.geometry.html).  When I attempt to read these via GeoPandas + Fiona, I end up with null geometries.  It is unclear to me if these are supported by the GeoJSON spec or if this is something not implemented by Fiona?

These look like so in GeoJSON
```json
"geometry":{"curveRings":[[[-643512,2228746],[-642912,2228695],...]]}
```

**Comment by sgillies** [comment]
@akrherz those geometry types are particular to Esri's software.
